flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2477,9 +2530,30 @@ class FMCMainDisplay extends BaseAirliners {
 
     //TODO: fix this functionality
     trySetPreSelectedClimbSpeed(s) {
+        const isNextPhase = this.currentFlightPhase === FmgcFlightPhases.TAKEOFF;
```

Comment:
According to FCOM the preselected speed cannot be modified on the MCDU once the phase is active, only by rotating the FCU knob.
<img width="705" alt="image" src="https://user-images.githubusercontent.com/9995998/116977907-1b5f1000-ad17-11eb-80aa-03ce8811293b.png">

Reply:
Yes and this is true, isNextPhase just ensures that the pre-sel speed I am about to modify is is the next one in order to ensure correct population of the simvar.

That variable has nothing todo whether a value can be changed or not.